docs/source/array-best-practices.rst

```diff
@@ -66,30 +66,29 @@ Note that if you provide ``chunks='auto'`` then Dask Array will look for a
 Avoid Oversubscribing Threads
 -----------------------------
 
-.. tip::
-
-   When using the ``distributed`` scheduler, the ``OMP_NUM_THREADS``,
-   ``MKL_NUM_THREADS``, and ``OPENBLAS_NUM_THREADS`` environment variables
-   are automatically set to ``1`` when using :ref:`nanny` workers.
-   This helps avoid oversubscribing threads in common cases.
-
```

Comment:
I think this tip or something close should be kept, probably here and in distributed documentation somewhere. It is a good thing to advise users about these configurations options and how they could help.

Reply:
the tip was about the behavior that’s going to be removed and should therefore be removed. the whole rest of the section is guidance about how to do it manually:

> [certain] libraries [are] multi-threaded, and need to be told to use only one thread explicitly. You can do this with […]

Reply:
What I mean here is that we should still keep some link to distributed docs, and maybe put some warning about the change and when it occured. So I would maybe keep a tip or a warning saying something like:

```
   When using the ``distributed`` scheduler, the ``OMP_NUM_THREADS``,
   ``MKL_NUM_THREADS``, and ``OPENBLAS_NUM_THREADS`` environment variables
   were **previously** set to ``1`` when using :ref:`nanny` workers.
   This is not done anymore, but it might be needed to help avoid oversubscribing 
   threads in common cases. See [link to distributed doc]() for how to set
   this behaviour by default